src/lib/mollie/api/client.coffee

```diff
@@ -87,9 +87,21 @@ module.exports = class Client
 				body += data.toString();
 			);
 			response.on("end", ->
-				callback(JSON.parse(body))
+        if body.length
+          parsedBody = {}
+          try
+              parsedBody = JSON.parse(body)
+          catch e
+              console.log(e)
```

Comment:
A side-effect like a `console.log` is not wanted here. This error needs to be handled correctly. The `try-catch` around the `JSON.parse` is a good idea though.